hoodie-common/src/main/java/com/uber/hoodie/common/table/log/block/HoodieAvroDataBlock.java

```diff
@@ -115,26 +115,7 @@ public static HoodieLogBlock getBlock(HoodieLogFile logFile,
 
     // 3. Write the records
     Iterator<IndexedRecord> itr = records.iterator();
-    while (itr.hasNext()) {
-      IndexedRecord s = itr.next();
-      ByteArrayOutputStream temp = new ByteArrayOutputStream();
-      Encoder encoder = EncoderFactory.get().binaryEncoder(temp, null);
-      try {
-        // Encode the record into bytes
-        writer.write(s, encoder);
-        encoder.flush();
-
-        // Get the size of the bytes
-        int size = temp.toByteArray().length;
-        // Write the record size
-        output.writeInt(size);
-        // Write the content
-        output.write(temp.toByteArray());
-        itr.remove();
-      } catch (IOException e) {
-        throw new HoodieIOException("IOException converting HoodieAvroDataBlock to bytes", e);
-      }
-    }
+    writerIndexedRecord(writer, output, itr);
```

Comment:
same here.. please just limit this PR to BucketedIndex removal

Reply:
> same here.. please just limit this PR to BucketedIndex removal

ok, I fix this content.